zerver/tests/test_message_notification_emails.py

```diff
@@ -297,18 +314,35 @@ def _extra_context_in_missed_stream_messages_stream_wildcard_mention_in_followed
         )
 
     def _extra_context_in_missed_stream_messages_topic_wildcard_mention(
-        self, show_message_content: bool = True
+        self,
+        show_message_content: bool = True,
+        receiver_is_participant: bool = True,
```

Comment:
It would probably be better to make these mandatory kwargs, rather than only having callers pass them when the value is false.

Reply:
Made `receiver_is_participant` a mandatory kwarg.

`show_message_content` remains as it is because the setting to whether include message content is True by default. So we only explicitly set it to false while testing that setting.

Reply:
OK.